client/rest-high-level/src/main/java/org/elasticsearch/client/core/PageParams.java

```diff
@@ -57,11 +57,11 @@ public PageParams(@Nullable Integer from, @Nullable Integer size) {
         this.size = size;
     }
 
-    public int getFrom() {
+    public Integer getFrom() {
```

Comment:
FYI, I did the same thing in feature branch:
https://github.com/elastic/elasticsearch/pull/41530/files